app/views/sessions/_card.html.erb

```diff
@@ -1,35 +1,62 @@
-<div class="w-full p-5 bg-white rounded-lg">
-    <div class="flex flex-row items-center mb-6">
-      <% if speaker.profile.image.attached? %>
-        <%= image_tag url_for(profile.image), class: "text-red w-12 h-12 rounded-full mr-4" %>
-      <% else %>
-        <div class="flex flex-row items-center justify-center w-12 h-12 p-2 mr-4 rounded-full min-w-12 min-h-12 bg-slate-300">
-            <%= image_tag url_for("user_avatar.svg") %>
+<div class="flex flex-col w-full p-5 mb-4 bg-white rounded-[20px]">
```

Comment:
I think we should start using [strict_locals](https://api.rubyonrails.org/classes/ActionView/Template.html#method-i-strict_locals-21) when using partials, as it makes them easier to reuse.